### PR TITLE
Rename derived to offline; simple renaming, no functionality changes

### PR DESCRIPTION
**Pull request by jimmarino**
Per our call on Monday, decision to change "derived" to "offline" in nomenclature.

### CODE REVIEW

File: extensions/auth/portability-auth-offline-demo/src/main/resources/META-INF/services/org.dataportabilityproject.spi.api.auth.extension.AuthServiceExtension [hunk 1]
```diff
@@ -0,0 +1 @@
+org.dataportabilityproject.auth.offline.DerivedDemoAuthServiceExtension
```
Comment (rtannenbaum) [inline extensions/auth/portability-auth-offline-demo/src/main/resources/META-INF/services/org.dataportabilityproject.spi.api.auth.extension.AuthServiceExtension]: missed a spot here?

File: extensions/auth/portability-auth-offline-demo/src/main/java/org/dataportabilityproject/auth/offline/DerivedDemoAuthServiceExtension.java [hunk 2]
```diff
@@ -24,15 +24,15 @@
...
 
 /**
- * Provides an extension that demonstrates how import of derived data can be implemented.
+ * Provides an extension that demonstrates how import of offline data can be implemented.
  *
- * <p>The extension receives demo derived data and simulates importing it by printing the data to
+ * <p>The extension receives demo offline data and simulates importing it by printing the data to
  * the console.
  */
 public class DerivedDemoAuthServiceExtension implements AuthServiceExtension {
```
Comment (rtannenbaum) [inline extensions/auth/portability-auth-offline-demo/src/main/java/org/dataportabilityproject/auth/offline/DerivedDemoAuthServiceExtension.java]: missed a spot here? (class name)

File: extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/dataportabilityproject/transfer/microsoft/MicrosoftTransferExtension.java [hunk 3]
```diff
@@ -24,18 +24,18 @@
   private static final String CONTACTS = "contacts";
   private static final String CALENDAR = "calendar";
   private static final String PHOTOS = "photos";
-  private static final String DERIVED_DATA = "derived-data";
+  private static final String DERIVED_DATA = "offline-data";
```
Comment (rtannenbaum) [inline extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/dataportabilityproject/transfer/microsoft/MicrosoftTransferExtension.java]: missed a spot here? (var name)

**Comment by jimmarino** [comment]
Thanks, I accidentally had match case on!

**Pull request merged**